api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/AllergyIntoleranceReactionComponentTranslatorImplTest.java

```diff
@@ -79,26 +77,58 @@ public void setUp() {
 	public void toFhirResource_shouldTranslateReactionToManifestation() {
 		Concept concept = new Concept();
 		concept.setUuid(CONCEPT_UUID);
+		String conceptName = "Test Reaction";
```

Comment:
We already have this value as a constant. I don't get why we'd create a new variable with the same value in this test.